x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/rollup/get_jobs.yml

```diff
@@ -103,8 +103,7 @@ setup:
 "Test get all jobs":
 
   - skip:
-      version: all
-      reason: Job ordering isn't guaranteed right now, cannot test
+      awaits_fix: "Job ordering isn't guaranteed right now, cannot test"
```

Comment:
Same as before: should we find/open an issue, for traceability? Or if not, remove the test?

Reply:
Proposal, once this PR is merged (keeping the tests), I follow up with new PRs deleting all of these muted tests and I'll tag respective people with the option to create a ticket / issue as alternative.

Reply:
I like it, giving people a chance to review it with a fallback of solving it for them (by deletion)